pkg/metrics/statsd/statsd.go

```diff
@@ -112,7 +112,12 @@ func (s *statsd) run() {
 }
 
 func (s *statsd) dial() (err error) {
-	s.conn, err = net.Dial("unix", s.env.MetricsSocketPath())
+	metricsSocketPath := "/var/etw/mdm_statsd.socket"
+	if s.env.Type() == env.Dev {
+		metricsSocketPath = "mdm_statsd.socket"
+	}
```

Comment:
personally i think this kind of thing all about the code base is harder to read/reason about than the current one
kind of reminds me of c code #ifdef madness